tortoisebeacon/config.go

```diff
@@ -9,66 +9,58 @@ import (
 
 // Config is the configuration of the Tortoise Beacon.
 type Config struct {
-	Kappa                    uint64        `mapstructure:"tortoise-beacon-kappa"`                       // Security parameter (for calculating ATX threshold)
-	Q                        *big.Rat      `mapstructure:"tortoise-beacon-q"`                           // Ratio of dishonest spacetime (for calculating ATX threshold). It should be a string representing a rational number.
-	RoundsNumber             types.RoundID `mapstructure:"tortoise-beacon-rounds-number"`               // Amount of rounds in every epoch
-	GracePeriodDuration      time.Duration `mapstructure:"tortoise-beacon-grace-period-duration"`       // Grace period duration
-	ProposalDuration         time.Duration `mapstructure:"tortoise-beacon-proposal-duration"`           // Proposal phase duration
-	FirstVotingRoundDuration time.Duration `mapstructure:"tortoise-beacon-first-voting-round-duration"` // First voting round duration
-	VotingRoundDuration      time.Duration `mapstructure:"tortoise-beacon-voting-round-duration"`       // Voting round duration
-	WeakCoinRoundDuration    time.Duration `mapstructure:"tortoise-beacon-weak-coin-round-duration"`    // Weak coin round duration
```

Comment:
Why do you want to remove and not split one round into voting part and weak coin part?

Reply:
i don't see any reason to split it in current implementation